Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3
```

Comment:
we can chose the best fit base image according to needs of our app setup however many times the lightest images are enough which are generally tagged as `:alpine` or `:<version>-alpine`.

Please read more about alpine images: https://nickjanetakis.com/blog/the-3-biggest-wins-when-using-alpine-as-a-base-docker-image

Also read the section *python:<version>-alpine* in official doc of Python docker image. https://hub.docker.com/_/python

---

The second point here is that instead of choosing the major version tag which in current case is `:3`, we should select a little more specific version especially in case of Python but also generally too. So far example, we are building our app using python version 3.6, we should use the image tag `:3.6`.

Later if we decide to upgrade the version, we'll adapt our app as well as the Docker configurations.